.gitignore

```diff
@@ -2,3 +2,7 @@ prepare_data/parameters.sh
 ivadomed.egg-info/
 ivadomed/__init__.pyc
 ivadomed/__pycache__/
+ivadomed/loader/__pycache__/
+ivadomed/FiLM/__pycache__/
```

Comment:
ahh right! Yes, I think a file film.py would make more sense since there is not much to put in the FiLM folder. For the models/film.py, since FiLM is integrated directly in the UNet, I am not sure a models/film.py would be necessary. Does it look good to you if I put the content of FiLM/utils.py in a file film.py (at the same level as main.py)?

Reply:
As the content of `FiLM/utils.py` deals with loading things, I was thinking within the loader folder. But, really, I am very okay with 'at the same level as main.py' too. Your call :)